pkg/build/api/validation/validation.go

```diff
@@ -123,13 +130,16 @@ func validateBuildSpec(spec *buildapi.BuildSpec, fldPath *field.Path) field.Erro
 	return allErrs
 }
 
-const maxDockerfileLengthBytes = 60 * 1000
+const (
+	maxDockerfileLengthBytes  = 60 * 1000
+	maxJenkinsfileLengthBytes = 60 * 1000
+)
```

Comment:
I assume so - if anyone wants to use anything bigger then they should commit to source repo (which will be the more common use case anyway).